jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/DefaultComparator.java

```diff
@@ -0,0 +1,55 @@
+/* ==========================================
+ * JGraphT : a free Java graph-theory library
+ * ==========================================
+ *
+ * Project Info:  http://jgrapht.sourceforge.net/
+ * Project Creator:  Barak Naveh (http://sourceforge.net/users/barak_naveh)
+ *
+ * (C) Copyright 2003-2008, by Barak Naveh and Contributors.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* -------------------------
+ * DefaultComparator.java
+ * -------------------------
+ * (C) Copyright 2015, by Fabian Späh and Contributors.
+ *
+ * Original Author:  Fabian Späh
+ * Contributor(s):   Rita Dobler
+ *
+ * $Id$
+ *
+ * Changes
+ * -------
+ * 20-Jun-2015 : Initial revision (FS);
+ *
+ */
+package org.jgrapht.alg.isomorphism;
+
+import java.util.Comparator;
+
+
+/**
+ * This is a default implementation for the check on semantic equality between
+ * vertices or edges.
+ */
+public class DefaultComparator<T>
```

Comment:
Since this class is not specific to isomorphism, let's put it in package org.jgrapht.util instead, and rename it to AlwaysEqualComparator to make its behavior clearer.